src/bento.js

```diff
@@ -11,12 +11,10 @@ if (!isEsm()) {
   installCustomElements(self, class {});
 }
 
-const waiting = self.BENTO || [];
-const push = (fn) => fn();
-
-// eslint-disable-next-line local/window-property-name
-self.BENTO = {push};
-
-for (let i = 0; i < waiting.length; i++) {
-  push(waiting[i]);
-}
+export * from '#core/context';
+export * from '#preact';
+export * from '#preact/base-element';
+export * from '#preact/compat';
+export * from '#preact/component';
+export * from '#preact/context';
+export * from '#preact/slot';
```

Comment:
This file was unused, I forgot to remove it when we generated it. But we use it now!